frontend/src/components/routingComponents/Routing.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import SignIn from '../../pages/SignIn';
+import SignUp from '../../pages/SignUp';
+import Home from '../../pages/Home';
+import Topics from '../../pages/Topics';
+import MyTeam from '../../pages/MyTeam';
+import Profile from '../../pages/Profile';
+import Nav from './Nav';
+import ProtectedRoute from './ProtectedRoute';
+import { setAuthenticationStatus } from '../../state/actions';
+
+const Routing = () => {
+  const [isAuthenticated, setAuthentificationStatus] = useState(false);
+
+  const dispatch = useDispatch();
+
+  const changeAuthentificationStatus = () => {
+    dispatch(setAuthenticationStatus(!isAuthenticated));
+    setAuthentificationStatus(!isAuthenticated);
+  };
+
+  return (
+    <Router>
+      <div>
+        {isAuthenticated ? <Nav /> : null}
+        <Switch>
+          <Route path="/" exact>
+            <SignIn onLogin={() => changeAuthentificationStatus()} />
```

Comment:
why do we have 2 different namings for the same thing (signin, login)?

Reply:
my mistake, fixed